pkg/apis/servicecatalog/validation/instance.go

```diff
@@ -185,6 +185,14 @@ func validateServiceInstanceStatus(status *sc.ServiceInstanceStatus, fldPath *fi
 func validateServiceInstancePropertiesState(propertiesState *sc.ServiceInstancePropertiesState, fldPath *field.Path, create bool) field.ErrorList {
```

Comment:
I think we should add a validation that ensures that if one of these fields are set, they both are.

Reply:
The both need to be set always. That is what the validation that I added does.

Reply:
Ack, I somehow misread that.